src/covid19sim/base.py

```diff
@@ -1118,14 +1128,14 @@ def update_book(self, human, date=None, risk_level = None):
 
     def send_message(self, owner, tracing_method, order=1, reason="test", payload=None):
         """
-        [summary]
+        Sends messages to all `Human`s in self.book.
 
         Args:
-            owner ([type]): [description]
-            tracing_method ([type]): [description]
-            order (int, optional): [description]. Defaults to 1.
-            reason (str, optional): [description]. Defaults to "test".
-            payload ([type], optional): [description]. Defaults to None.
+            owner ([Human]): owner of this contact book.
```

Comment:
I'm sorry to be this annoying person @pg2455 but type docs should be

```
owner (Human): owner of this contact book.
```

For reference, here's a complete example: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Reply:
so annoying ... 💢 👎

Reply:
done 👍